pymongo/ocsp_support.py

```diff
@@ -30,6 +30,12 @@
 )
 from cryptography.hazmat.primitives.asymmetric.padding import PKCS1v15 as _PKCS1v15
 from cryptography.hazmat.primitives.asymmetric.rsa import RSAPublicKey as _RSAPublicKey
+from cryptography.hazmat.primitives.asymmetric.x448 import (
+    X448PublicKey as _X448PublicKey,
```

Comment:
This is new in cryptography 2.5 so we would need to bump our min version: https://cryptography.io/en/latest/hazmat/primitives/asymmetric/x448/#cryptography.hazmat.primitives.asymmetric.x448.X448PublicKey

We have:
```
.evergreen/test-encryption-requirements.txt:2:cryptography>=2
```

And also in pymongocrypt:
```
bindings/python/setup.py:58:        "cryptography>=2.0",
```

Reply:
We might need to add cryptography>2.5 as an explicit dependency for OCSP.

Reply:
Is bumping cryptography to >2.5 acceptable for OCSP?

Reply:
It should be fine given the version reqs here: https://github.com/pyca/pyopenssl/blob/17.2.0/setup.py#L99

Reply:
Either this or you could try/except importerror and add a fallback.

Reply:
So we don't need any changes for anything but the test-encryption-requirements + pymongocrypt?

Reply:
I see two options:
1) add cryptography>=2.5 as an explicit dependency for the OCSP extra. Also update test-encryption-requirements and pymongocrypt.
2) add a try/except importerror and add a fallback for this import (and maybe the other one)

Reply:
Option 1 seems cleaner to me, especially since 2.0 is over 5 years old.

Reply:
SGTM! could you also mention this dependency change in the changelog?

Reply:
Done, and PR opened for pymongocrypt change: https://github.com/mongodb/libmongocrypt/pull/668